catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/suggestion/LatLonCoordinateProcessorTest.java

```diff
@@ -253,19 +250,24 @@ public void testRegexIgnoresStandaloneSymbols() {
   @Test
   public void testRegexIgnoresStandaloneSymbolsChainedTwice() {
     List<Suggestion> suggestions =
-        processor.enhanceResults(new LinkedList<>(), "blah10.19205123gagne .. -- -.12 . 12 - 14");
+        enhanceAndReturn(new LinkedList<>(), "blah10.19205123gagne .. -- -.12 . 12 - 14");
     assertThat(
         cast(suggestions).get(0), hasCoordinates(from(10.19205123, -0.12), from(12.0, 14.0)));
   }
 
   @Test
   public void testRegexIgnoresStandaloneSymbolsChainedThrice() {
     List<Suggestion> suggestions =
-        processor.enhanceResults(new LinkedList<>(), "blah10.19205123gagne ... --- -.12 . 12 - 14");
+        enhanceAndReturn(new LinkedList<>(), "blah10.19205123gagne ... --- -.12 . 12 - 14");
     assertThat(
         cast(suggestions).get(0), hasCoordinates(from(10.19205123, -0.12), from(12.0, 14.0)));
   }
 
+  private List<Suggestion> enhanceAndReturn(List<Suggestion> input, String query) {
```

Comment:
Maybe it would be better to initialize the LinkedList of suggestions yourself and prepend processor results? (you would have to change the void enhanceResults)

Reply:
https://github.com/codice/ddf/pull/3901#discussion_r227936642

Tried to write the method calls around something that works with `QueryApplication` in a non-intrusive way. I think that's the priority. After that, the tests can do whatever they need to make their lives easier.